app/models/race_entry.rb

```diff
@@ -27,7 +27,7 @@ def default_values
   end
 
   def elapsed_predicted_time
-    predicted_time && ChronicDuration.output(predicted_time) || 'NA'
+    predicted_time && ChronicDuration.output(predicted_time)
```

Comment:
This comment is not about the change here, but just about what this time does. What is an elapsed_predicted_time? It's not just the predicted_time? It clearly isn't an integer, since before this change it could return a string. So, it must be a string. Is it just the string representation of the predicted_time?